src/app/App.js

```diff
@@ -2,11 +2,13 @@ import React from 'react';
 import './App.css';
 import ContactForm from '../contact/ContactForm';
 import SignInForm from '../user/SignInForm';
+import SignUpForm from '../user/SignUpForm';
```

Comment:
Change SignUp to Register
```suggestion
import RegisterForm from '../user/RegisterForm';
```

Reply:
Fixed up! :)